src/main/java/com/Incubatormsrv/Incubatormsrv/OkHttpGet.java

```diff
@@ -0,0 +1,33 @@
+package com.Incubatormsrv.Incubatormsrv;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+import okhttp3.ResponseBody;
+import retrofit2.Call;
+import retrofit2.Callback;
+import retrofit2.Retrofit;
+import retrofit2.converter.gson.GsonConverterFactory;
+
+import java.io.IOException;
+
+public class OkHttpGet {
```

Comment:
This isn't correct way to create your interfaces for injecting an interface. Please read through this and redo this task. It's very far off the correct approach. 

Essentially
- You need a service interface that is generated from swagger along with the models (Weather service)
- Your controller endpoint definitions also need to be generated swagger (Controller Spring rest controller interface)
- generation for your controller vs the API call is different in terms of the pom configuration
- All your stuff needs to be configurable, so the url below needs to be configurable.
- 

https://www.baeldung.com/spring-boot-rest-client-swagger-codegen